### PR TITLE
Access-Sweden project

### PR DESCRIPTION
**Pull request by docNord**
First draft access sweden project. Please review and suggest changes @PeterTorstensson

### CODE REVIEW

File: content/posts/access-sweden-2023-04-05.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+title: Access Sweden - ett nytt samhällskontrakt för hållbart resande utan bil
+date: 2023-04-05
+modified:
+author: Anders Nord
+category: Projekt
+tags: cykel, kollektivtrafik, forskning, trafikplanering, 2030
+data:
+status: unpublished
+
+**Vinnova har godkänt stöd till forskningsprojektet Access Sweden, med Statens
```
Comment (PeterTorstensson) [inline content/posts/access-sweden-2023-04-05.md:10]: Ändra till: **Vinnova, Energimyndigheten och Formas har godkänt stöd till ett förberedelseprojektet inom Impact innovation med titel Access Sweden. Statens

File: content/posts/access-sweden-2023-04-05.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Väg- & Transportforskningsinstitut (VTI) som huvudman. Projektet har som uppgift
```
Comment (PeterTorstensson) [inline content/posts/access-sweden-2023-04-05.md:11]: Ändra till: Väg- & Transportforskningsinstitut (VTI) är huvudman med VREF och RISE som övriga projektdeltagare. Projektets syfte är

File: content/posts/access-sweden-2023-04-05.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+att utveckla ett nytt slagkraftigt innovationsprogram för att stötta samhället
+med rörlighet utan tillgång till egen bil. IFKs ordförande Anders Nord är med
```
Comment (PeterTorstensson) [inline content/posts/access-sweden-2023-04-05.md:13]: Ändra till: med hållbar tillgänglighet utan att behöva äga egen bil. IFKs ordförande Anders Nord är med

File: content/posts/access-sweden-2023-04-05.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+som en av många tekniska rådgivare.**
```
Comment (PeterTorstensson) [inline content/posts/access-sweden-2023-04-05.md:14]: Ändra till: som en av många rådgivare.**

File: content/posts/access-sweden-2023-04-05.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Projektet Access Sweden beskrivs i detalj genom <a href="https://www.radslaget.se/"
```
Comment (PeterTorstensson) [inline content/posts/access-sweden-2023-04-05.md:16]: För en beskrivning av Access Sweden kan vi länka till detta pressmeddelande: https://www.radslaget.se/pressmeddelande

File: content/posts/access-sweden-2023-04-05.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+target="_blank">Rådslaget</a>, ett initiativ från nationellt kunskapscentrum för
```
Comment (PeterTorstensson) [inline content/posts/access-sweden-2023-04-05.md:17]: Det är lätt att blanda samman Rådslaget med Access Sweden, men Rådslaget är ett tidigare utfört forskningsprojekt. Kärngruppen till Rådslaget är ungefär densamma som till Access Sweden och arbetsmetoderna med rådgivargrupp och workshops har ärvts (som sagt, till förvillelse likt). Jag föreslår att stryka rad 17-21.

**Comment by docNord** [comment]
We are discussing the label here - @PeterTorstensson  points out that "Projekt" may indicate that we are responsible for the project, when in fact we are only part of the advisory board. Could we add an additional label to cover this type of activities, and what should we call them - any ideas @mgcth ?

**Comment by mgcth** [comment]
You can put it under a separate category (just type that category) and It'll be listed under `Akriv`. Do you want a separate menu button? That is also possible to add, but we need to include a dropdown menu for mobile screens, in that case.

**Comment by mgcth** [comment]
Could it be filed under a separate subtitle in Om, perhaps?

**Comment by mgcth** [comment]
Or it could also go under the home page, for example, if you don't want to add too many pages. It makes sense to keep it as simple as possible, for now.

**Comment by docNord** [comment]
I suppose that could work, but I would rather Om presented who we are and then we had what we do in other headlines: Events and Projekts. Given those are our options, Projekt is closest, IMHO. @PeterTorstensson , what are your thoughts on this?

**Comment by PeterTorstensson** [comment]
I like us to be clear on IFKs engagement in this project since I am the one that has invited us/Anders to participate. It would be embarrassing if someone perceives that IFK is exaggerated in its communication regarding this engagement.

I agree with Anders regarding the Om-alternative, I would prefer another solution.  

I think Access Sweden can remain in the Projekt-category if we change the name of the project to e.g. “advisor-access-sweden”. What do you think – could this maybe be a way forward?

**Comment by docNord** [comment]
Shall we try to get this out @PeterTorstensson ? It would be nice to show some activity on the website. I am also working on the material I have from the interview with Sasha, but this is already finished and so could go up on the website as soon as we agree on the final discussion points. :)

**Review by PeterTorstensson — APPROVED** [review]

**Review by mgcth — APPROVED** [review]

**Pull request merged**